src/sa_web/static/js/views/app-view.js

```diff
@@ -89,9 +91,11 @@ var Shareabouts = Shareabouts || {};
       });
 
       // Handle collection events
-      this.collection.on('add', this.onAddPlace, this);
-      this.collection.on('remove', this.onRemovePlace, this);
-
+      // REFACTOR
+      // bind add and remove events explicity to the places property of the collections object:
+      this.collection.places.on('add', this.onAddPlace, this);
```

Comment:
I'm sure this is on your todo list, but we should definitely abstract this to either iterate through all places instead of hard-coding the places key.